keras_contrib/applications/densenet.py

```diff
@@ -331,6 +338,9 @@ def __transition_block(ip, nb_filter, compression=1.0, dropout_rate=None, weight
                     in the transition block.
         dropout_rate: dropout rate
         weight_decay: weight decay factor
+        dilation_rate: an integer or tuple/list of 2 integers, specifying the dilation rate to
+        use for dilated convolution. Can be a single integer to specify the same value for
+        all spatial dimensions.
 
```

Comment:
The docstring should also explain pooling parameter. Just copy paste from the DenseNet docstring.